<?xml version="1.0" ?>

<launch>    
	
    <param name="offset_path"     value="$(find thormang3_manager)/config/offset.yaml"/>	
    <!-- <arg name="offset_path"  value="$(find thormang3_manager)/config/offset.yaml"/> -->
    <!-- <param name="offset_table"      value="$(arg offset_path)"/> -->
    <!-- <rosparam command="load"        file="$(arg offset_path)" /> -->
 	<param name="robot_file_path" value="$(find thormang3_manager)/config/THORMANG3.robot"/>
    <param name="init_file_path"  value="$(find thormang3_manager)/config/dxl_init.yaml"/>   
    	
    <node name="offset_tuner_server_node" pkg="thormang3_offset_tuner_server" type="offset_tuner_server_node" output="screen">
    </node>
    
</launch>
